src/SDK/Metrics/MeterProvider.php

```diff
@@ -117,4 +117,9 @@ public function forceFlush(): bool
 
         return $success;
     }
+
+    public function builder(): MeterProviderBuilder
```

Comment:
Should this be a static method? That's how it's accessed in the `sdk_builder.php` example.

See https://github.com/open-telemetry/opentelemetry-php/pull/845/files#diff-6abbe36ded0213fbcc94f09c6e4c6ab26180913a32a73db97533baeed94c95a1R39

Reply:
yes it should be static, thanks for picking that up :)